# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rpcs3/hotfix-llvm-17.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- rpcs3-3067c86d658e68f41452ea75e894d0149d685017/3rdparty/llvm/CMakeLists.txt.vanilla	2024-02-23 02:19:43.836004202 +0000
+++ rpcs3-3067c86d658e68f41452ea75e894d0149d685017/3rdparty/llvm/CMakeLists.txt	2024-02-23 02:28:29.436042977 +0000
@@ -53,10 +53,7 @@
 			set(LLVM_DIR ${CMAKE_SOURCE_DIR}/${LLVM_DIR})
 		endif()
 
-		find_package(LLVM 16.0 CONFIG)
-		if(NOT LLVM_FOUND)
-			find_package(LLVM 17.0 CONFIG)
-		endif()
+		find_package(LLVM 17.0 CONFIG)
 
 		if (NOT LLVM_FOUND)
 			if (LLVM_VERSION AND LLVM_VERSION_MAJOR LESS 16)
